docker-images/build.sh

```diff
@@ -63,20 +65,67 @@ function build {
     # Images depending on Kafka version (possibly indirectly thru FROM)
     for kafka_version in "${!version_checksums[@]}"
     do
-        sha=${version_checksums[$kafka_version]}
+        echo "Creating container image for Kafka $kafka_version"
+
+        expected_sha=${version_checksums[$kafka_version]}
         lib_directory=${version_libs[$kafka_version]}
+
+        binary_file_dir="./$kafka_image/tmp"
+
+        # If there is a file specified for this version of Kafka use that instead of the specified URL
+        if [ ${version_binary_files[$kafka_version]} ]
+        then 
+
+            custom_binary_file_path=${version_binary_files[$kafka_version]}
+            binary_file_name=$(basename "$custom_binary_file_path")
+            binary_file_path="$binary_file_dir/$binary_file_name"
+            echo "Kafka binary file field was specified, using $custom_binary_file_path instead of URL"
+
+            if [ -f "$binary_file_path" ]
+            then
+                echo "$binary_file_name already present in build directory"
+            else
+                echo "Copying $binary_file_name to build directory"
+                cp "$custom_binary_file_path" "$binary_file_path" 
```

Comment:
Will thsi work on MacOS? You should better use the multiplatform support scripts here so that this works also on MacOS etc.

Reply:
A very good point, I had missed that.